docs/using-modules.rst

```diff
@@ -169,14 +171,19 @@ Sometimes, you may encounter a module which itself ``uses`` other modules. Vyper
     # export all external functions from ownable_2step
     exports: ownable_2step.__interface__
 
+.. note::
```

Comment:
I think we could replace this with `warning::` since the context justifies it imo.

Reply:
18f6a817e0902b1a41d07692e5721af49e5f5db2